drivers/gpu/drm/v3d/v3d_drv.c

```diff
@@ -198,6 +198,36 @@ static const struct of_device_id v3d_of_match[] = {
 };
 MODULE_DEVICE_TABLE(of, v3d_of_match);
 
+static void
+v3d_idle_sms(struct v3d_dev *v3d)
+{
+	if (v3d->ver < 71)
```

Comment:
Shouldn't this (and others) be `V3D_GEN_71`?

Reply:
It is not needed. In the end, `V3D_GEN_71 = 71` by the enum definition. But I can change it if you believe it will ease the code comprehension.

Reply:
Done, thanks for the feedback, I'll apply it in the upstream series as well.